gradle/libs.versions.toml

```diff
@@ -5,13 +5,13 @@ com-squareup-moshi = "1.15.1"
 com-squareup-okio = "3.9.1"
 de-mannodermaus-junit5 = "1.6.0"
 graalvm = "22.3.2"
-kotlinx-serialization = "1.6.3"
-ksp = "1.9.24-1.0.20"
+kotlinx-serialization = "1.7.3"
```

Comment:
Pretty major version bumps. Seems appropriate for OkHttp 5.

Reply:
Yup